doc/level1.rst

```diff
@@ -878,12 +878,43 @@ Level 1 MUAs MUST allow the user to disable Autocrypt completely for
 each account they control.  For level 1, we expect most MUAs to have
 Autocrypt disabled by default.
 
-Onboarding
-++++++++++
 
-.. todo::
+Helping Users getting Started
++++++++++++++++++++++++++++++
+
+The following paragraph provides recommendations for MUA
+implementations to help users start Autocrypt after an
+account was set up.
+
+The MUA SHOULD scan the mailbox for sent mails (wherever
+they might be) that show evidence of OpenPGP/Autocrypt
+usage.
+
+
```

Comment:
Is the series below a "first-match-wins" situation?  or should the MUA try them all in parallel?

If the search takes a long time, do we need to exhaust the search first, and then decide what to do based on what we find, or should we take these actions as we find stuff?